app/scenes/Talk/components/Pane/index.js

```diff
@@ -23,7 +23,9 @@ import Avatar from '../../../../components/Avatar';
 
 import Preview from '../Preview';
 
-function Speaker({ data, onPress }) {
+const isAndroid = Platform.OS === 'android';
```

Comment:
Kept redoing this check all over the place. I don't think most people change platforms while using the app 😝